pkgs/top-level/python-packages.nix

```diff
@@ -14788,6 +14788,27 @@ in modules // {
     };
   };
 
+  pyrr = buildPythonPackage rec {
+    name = "pyrr-${version}";
+    version = "0.7.0";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/pyrr/pyrr-${version}.tar.gz";
+      sha256 = "ddfcd83394e9703138a3c668747df3548a81238d8ab36d4bf6a84d25ea90b2cd";
+    };
+
+    # ImportError: No module named tests
+    doCheck = false;
+
+    buildInputs = with self; [ multipledispatch numpy ];
```

Comment:
Are those packages required at runtime? If so, they should be `propagatedBuildInputs`